src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,354 @@
+                                                                                        {props.sampleManager!.getComponentForSample(sampleId, 1, '')}
+                                                                                    </span>
+                                                                                ))}
+                                                                            </li>
+                                                                        ))}
+                                                                        </ul>
+                                                                    </Then>
+                                                                    <Else>
+                                                                        <span>{genomicGroupMatch.matches[0].trueHugoSymbol + ' ' + genomicGroupMatch.matches[0].trueProteinChange + ' '}
+                                                                            {genomicGroupMatch.matches[0].sampleIds.map((sampleId: string) => (
+                                                                                <span className={styles.genomicSpan}>
+                                                                                    {props.sampleManager!.getComponentForSample(sampleId, 1, '')}
+                                                                                </span>
+                                                                            ))}
+                                                                        </span>
+                                                                    </Else>
+                                                                </If>
+                                                            </span>
+                                                        </div>
+                                                    </Else>
+                                                </If>
+                                            ))}
+                                            <If condition={clinicalGroupMatch.notMatches.length > 0}>
+                                                <DefaultTooltip
+                                                    placement='bottomLeft'
+                                                    trigger={['hover', 'focus']}
+                                                    overlay={this.tooltipGenomicContent(clinicalGroupMatch.notMatches)}
+                                                    arrowContent={<div className="rc-tooltip-arrow-inner" />}
+                                                    destroyTooltipOnHide={false}
+                                                    onPopupAlign={placeArrowBottomLeft}
+                                                >
+                                                    {this.mainContent('No alterations in ' + this.getHugoSymbolName(clinicalGroupMatch.notMatches, 3) + ' defined by the trial')}
```

Comment:
Let's move this logic to a util function and test.

Reply:
@zhx828 This function contains html tag. Can we move it to utils?